blockchain/chaingen/generator.go

```diff
@@ -2332,6 +2364,16 @@ func (g *Generator) AssertTipHeight(expected uint32) {
 	}
 }
 
+// AssertLiveTickets panics if the current tip block associated with the generator
+// has an empty ticket pool.
+func (g *Generator) AssertLiveTickets() {
+	poolSize := g.tip.Header.PoolSize
+	if poolSize == 0 {
+		panic(fmt.Sprintf("pool size for block %q is 0 "+
```

Comment:
The grammar isn't quite right here. I think it's perfectly clear to just leave it at `pool size for block %q is 0`.